types/index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for Chalk
+// Definitions by: Thomas Sauer <https://github.com/t-sauer>
+
+export = chalk;
+
+declare function chalk(...text: string[]): string;
+declare function chalk(text: TemplateStringsArray, ...placeholders: string[]): string;
+
+declare namespace chalk {
+
+	export enum ChalkLevel {
+		Disabled = 0,
```

Comment:
I would call it `None` as it's not technically disabled, but rather no color support.